docs/bigquery/snippets.py

```diff
@@ -255,41 +255,66 @@ def test_list_tables(client, to_delete):
     dataset = client.create_dataset(dataset)
     to_delete.append(dataset)
 
-    # [START list_tables]
+    # [START bigquery_list_tables]
     tables = list(client.list_tables(dataset))  # API request(s)
     assert len(tables) == 0
 
     table_ref = dataset.table('my_table')
     table = bigquery.Table(table_ref)
-    table.view_query = QUERY
-    client.create_table(table)                          # API request
+    client.create_table(table)                  # API request
     tables = list(client.list_tables(dataset))  # API request(s)
```

Comment:
We should double-check with the BigQuery team to see if table listing is strong consistent or eventually consistent. If eventually consistent, this test / sample could be flakey.